app/assets/stylesheets/cart.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Cart controller here.
```

Comment:
delete empty file (not gonna comment on the rest of the scss ones but they should all go)